pastas/modelstats.py

```diff
@@ -497,6 +497,34 @@ def diagnostics(
         stats: tuple = (),
         float_fmt: str = "{0:.2f}",
     ) -> DataFrame:
+        """Methods to compute various diagnostics checks for a time series.
```

Comment:
Methods to compute various diagnostics checks for the noise time series. If no NoiseModel is used, the diagnostics are computed on the model residuals.